tests/nested/manual/muinstaller-real/task.yaml

```diff
@@ -241,9 +249,18 @@ execute: |
       local snap_new_dir=$3
 
       printf "Test installing snap from file %s\n" "$snap_name"
-      fakestore make-refreshable --dir "$STORE_DIR" --snap-orig-blob "$snap_orig_blob" --snap-blob "$snap_new_dir" "$snap_name"
+      fakestore make-refreshable --dir "$STORE_DIR" \
+          --snap-orig-blob "$snap_orig_blob" --snap-blob "$snap_new_dir" "$snap_name"
+      remote.exec mkdir -p asserts/
+      # Order of assertion acking is important
+      for p in "$STORE_DIR"/asserts/*.account-key "$STORE_DIR"/asserts/*.snap-declaration "$STORE_DIR"/asserts/*.snap-revision; do
+          remote.push "$p" asserts/
+          remote.exec sudo snap ack asserts/"$(basename "$p")"
+      done
+      remote.push "$STORE_DIR"/"$snap_name"_*.snap
+
       boot_id=$(tests.nested boot-id)
-      REMOTE_CHG_ID=$(remote.exec sudo snap refresh --amend --no-wait --channel="$version/edge" "$snap_name")
+      REMOTE_CHG_ID=$(remote.exec "sudo snap install --no-wait ${snap_name}_*.snap")
```

Comment:
this probably needs a comment why we are using a local snap install with assertions, we might not need the fakestore if we do this as we could just push and ack the assertions instead?